script/build.js

```diff
@@ -117,10 +117,12 @@ if (options.buildtype === 'production') {
   ignoreList.push('profile/*');
   ignoreList.push('auth/*');
   ignoreList.push('education/apply-for-education-benefits-new');
+  ignoreList.push('education/index-new');
 } else if (options.buildtype === 'staging') {
   ignoreList.push('facilities/*');
   ignoreList.push('profile/*');
   ignoreList.push('education/apply-for-education-benefits-new');
+  ignoreList.push('education/index-new');
```

Comment:
Good point, guess it doesn't hurt! Since it's all password protected anyway. Probably for the page above as well. I will fix that.